components/Form.js

```diff
@@ -23,13 +22,22 @@ export default function Form({
     let results = [];
 
     if (data.activity && data.region === "disabled") {
-      results = places.filter((place) => place.activity === data.activity);
+      results = places.filter((place) => {
+        if (Array.isArray(place.activity)) {
+          return place.activity.includes(data.activity);
+        } else {
+          return place.activity === data.activity;
+        }
```

Comment:
I think `place.activity` will always be an array, so this should be fine:

```suggestion
          return place.activity.includes(data.activity);
```